### PR TITLE
Assorted code style fixes

### PR DESCRIPTION
**Pull request by mcostalba**
No functional change.

### CODE REVIEW

File: src/search.cpp [hunk 1]
```diff
@@ -308,13 +312,10 @@ void MainThread::search() {
...
-// Sizes and phases of the skip-blocks, used for distributing search depths across the threads.
-static int skipsize[20] = {1, 1, 2, 2, 2, 2, 3, 3, 3, 3, 3, 3, 4, 4, 4, 4, 4, 4, 4, 4};
-static int phase   [20] = {0, 1, 0, 1, 2, 3, 0, 1, 2, 3, 4, 5, 0, 1, 2, 3, 4, 5, 6, 7};
 
-// Thread::search() is the main iterative deepening loop. It calls search()
-// repeatedly with increasing depth until the allocated thinking time has been
-// consumed, the user stops the search, or the maximum search depth is reached.
+/// Thread::search() is the main iterative deepening loop. It calls search()
+/// repeatedly with increasing depth until the allocated thinking time has been
+/// consumed, the user stops the search, or the maximum search depth is reached.
```
Comment (loco-loco) [inline src/search.cpp]: Is there a reason for the extra "/" ?

**Comment by mcostalba** [comment]
This is our usual convention for functions outside local namespace (global visibility)

File: src/search.cpp (hunk 1 above)
Comment (mcostalba) [inline src/search.cpp]: This is our usual convention for functions outside local namespace (global visibility)

File: src/search.cpp [hunk 2]
```diff
@@ -350,16 +351,19 @@ void Thread::search() {
...
   // Iterative deepening loop until requested to stop or the target depth is reached
   while (   (rootDepth += ONE_PLY) < DEPTH_MAX
          && !Signals.stop
          && (!Limits.depth || Threads.main()->rootDepth / ONE_PLY <= Limits.depth))
   {
-      // skip half of the plies in blocks depending on game ply and helper index.
-      if (idx && ((rootDepth / ONE_PLY + rootPos.game_ply() + phase[hIdx]) / skipsize[hIdx]) % 2)
-          continue;
+      // Distribute search depths across the threads, picking the next depth to search
+      // for each thread in a way to make each thread searching on a different depth.
```
Comment (pb00068) [inline src/search.cpp]: @mcostalba I find the sentence 'to make each thread searching on a different depth.' a bit misleading: it is not true that each thread will searching on a different (root)-depth, in fact often a part of them will do search with same rootDepth.  The principle is actually that each singular rootDepth is searched by exactly the half of helper-threads while the other half will skip it.

**Comment by Luvax** [comment]
lmao gl

**Pull request closed without merging**